moodle/moodlecontroller.go

```diff
@@ -153,7 +156,7 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	}
 
 	glog.Info("Starting workers")
-	// Launch two workers to process Foo resources
+	// Launch two workers to process Pod resources
```

Comment:
Launch two workers to process Moodle resources

Reply:
done